pkg/aws/iam.go

```diff
@@ -190,3 +196,27 @@ func unMarshallPolicyDocument(document string) (*PolicyDocument, error) {
 	}
 	return &policyDocument, nil
 }
+
+// getIAMRole returns the role object struct that has attributes of a given role
+func getIAMRole(ctx context.Context, client IAMClient, roleName string) (output *iam.GetRoleOutput, err error) {
+	input := &iam.GetRoleInput{
+		RoleName: &roleName,
+	}
+
+	output, err = client.GetRole(ctx, input)
+	if err != nil {
+		return output, err
+	}
+	log.Println(output.Role)
+
+	return output, err
+}
+
+// Asserts the MaxSessionDuration attribute of a given IAM Role
+func AssertIamRoleMaxSessionDuration(t *testing.T, ctx context.Context, client IAMClient, roleName string, maxDuration int32) {
+	output, err := getIAMRole(ctx, client, roleName)
+	assert.Nil(t, err)
```

Comment:
**Suggestion:** Rather than this, I would have the logic fail the test then exit the method. The reasons for this is that if the getIAMRole method fails with an error, it's very likely the `output` object will have a `nil` value, in which case all sorts of things could happen that would throw off the output of the tests. 
Note: please run `go fmt` on this, GitHub's suggestion formatting is not ideal.
```suggestion
	if err != nil {
	    t.Error(err)
	    return
	}
```

Reply:
done